### ISSUE TITLE
Configure Mssql connectionIsolationLevel to SNAPSHOT

### ISSUE THREAD

**Issue by lixaotec** [typeorm/typeorm#6153]
**Issue type:**

[x] question
[ ] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[x] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[ ] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[x] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

Dear fellows.

How to setup in json configs the isolationlevel for transaction and for connection without transaction to SNAPSHOT?

Thanks

**Comment by OliverLeighC** [comment]
I was wondering about this as well. Are there plans to support SNAPSHOT isolation level for mssql transactions?

looks like it wasn't included in the initial release https://github.com/typeorm/typeorm/pull/2227

**Comment by OliverLeighC** [comment]
@lixaotec Not sure if you've figured this out already but you can specify it on the connection https://github.com/typeorm/typeorm/blob/master/docs/connection-options.md

so for example: 
```
const { SNAPSHOT } = require("tedious").ISOLATION_LEVEL;

{
    name: "ConnectionName",
    type: "mssql",
    password: '',
    options: {
      isolationLevel: SNAPSHOT,
      connectionIsolationLevel: SNAPSHOT,
    },
    logging: true,
    maxQueryExecutionTime: 3000
}
```

**Comment by lixaotec** [comment]
Dear @OliverLeighC . thanks for helping.

In fact I did that, back at that time, however forgot to give a feed back.

Im going to close this issue, as I cannot recall what didnt worked at that time, I should have provided more info to become clear. Now even myself cant recall the exact issue (but there is one).

So Im very grateful for those who collaborated in my issue.

Btw, I recall another issue I´ve been at that moment 
https://github.com/typeorm/typeorm/issues/6161

It was probably for that I had tried to setup SNAPSHOT mode, as my database is configured with snapshot. The issue with deadlocks still remains, therefore I didnt advanced with that for now. 

Later ill try to bring the deatils of my other issue.

Best
